server/routes/users/user.controller.js

```diff
@@ -56,6 +64,58 @@ export const createUser = async (req, res) => {
 			const id = user.user_id;
 			await resetUserPassword(id);
 			await assignRolesToUser(id, roles);
+			const query = {
+				text: "INSERT INTO person (auth0_id, full_name, email, roles, country) VALUES ($1, $2, $3, $4, $5) ON CONFLICT (email) DO UPDATE SET auth0_id = $1, full_name = $2, email = $3, roles = $4, country = $5",
+				values: [id, fullName, email.toLowerCase(), roles.join(","), country],
+			};
+			await pool.query(query);
+			res.json(user);
+		})
+		.catch((err) => {
+			logger.error(err);
+			res.status(500).json({ message: err.message });
+		});
+};
+
+export const updateUser = async (req, res) => {
+	const { full_name, email, roles, country, auth0_id, allRoles, blocked } =
+		req.body;
+
+	const user = {
+		name: full_name,
+		email,
+		blocked,
+		user_metadata: {
+			country,
+		},
+		connection: config.AUTH0_CONNECTION,
+	};
+
+	await auth0
+		.updateUser({ id: auth0_id }, user)
+		.then(async (user) => {
+			await auth0.removeRolesFromUser({ id: auth0_id }, { roles: allRoles });
+			if (roles.length > 0) {
+				await assignRolesToUser(auth0_id, roles);
+			}
+			const query = {
+				text: "INSERT INTO person (auth0_id, full_name, email, roles, country, blocked) VALUES ($1, $2, $3, $4, $5, $6) ON CONFLICT (email) DO UPDATE SET auth0_id = $1, full_name = $2, email = $3, roles = $4, country = $5, blocked = $6",
```

Comment:
Could you please change this INSERT Into a Template Literal
So that it can been seen on a page without scrolling.
Thank you

Reply:
Template literals do not have sql sanitisation, we should not be using that and I'm using a query object, which gets messy when bringing it into multiple lines.